### DISCUSSION TITLE
Kagi Translation API

### DISCUSSION THREAD

**Discussion by tinusg** [kagisearch/kagi-docs#592] [Kagi Search API Feature Requests & Ideas]
I'm a DeepL API customer and would love a high quality alternative. Any plans for offering one?

**Comment by vprelovac** [comment]
We do not have one yet, but stay tuned.

**Comment by brycepg** [comment]
Let us know when you do please! I know this would require some kind of credits system rather than the $/month that's in place now. Would love to benchmark it against deepl/gt when you do.

**Comment by vprelovac** [comment]
We are ready to start taking beta testers for the API. please email luis@kagi.com with details (use cases, estimated monthly volume).

We are targetting the price of $15/million input chars. The quality you can already test out at https://translate.kagi.com

Thanks!

**Comment by shred** [comment]
Is proofreading part of the API?

**Comment by briana-brady** [comment]
I would love access to the dictionary features of Kagi Translate. Hands down an amazing resource for learning already. Taking it to the next level through API access would be great!

**Comment by jvican** [comment]
Hi @vprelovac. Sent an email to Luis on being a beta tester for the Kagi translate API but didn't hear back. Just a friendly ping - I'd love to sign up for it to beta test it, I'm already a big power user of Kagi Translate today and want to build a few projects.

**Reply by giorgiobrullo** [reply]
Hey, please send us an email at support@kagi.com and we'll get back with an API key :)

**Reply by orschiro** [reply]
Initially I also emailed Luis but never heard back either. I just forwarded my email to support. Looking forward to testing an API key for my [Vocabulary Sheet](https://orzanna.de/language/) project. :-)

**Comment by jessfdm-codes** [comment]
Did anything come of this? Is it out of beta, still on-going or abandoned?

**Reply by giorgiobrullo** [reply]
Hey, we are still giving out API keys in beta. Feel free to reach out at support@kagi.com if you'd like access.
